wrangler.toml

```diff
@@ -1,4 +1,4 @@
-name = "your-plugin-name"
+name = "issue-comment-embeddings"
```

Comment:
```suggestion
name = "ubiquibot-issue-comment-embeddings"
```

Reply:
I think we should prefix all with `uos-` because ubiquibot is deprecated.

Reply:
I wonder if it makes sense to also rename the repositories to match.